arviz/plots/compareplot.py

```diff
@@ -22,33 +25,37 @@ def plot_compare(
     show=None,
 ):
     """
-    Summary plot for model comparison.
+    Summary plot for model comparison
 
-    This plot is in the style of the one used in the book Statistical Rethinking (Chapter 6)
-    by Richard McElreath.
+    Models are compared based on their expected log pointwise predictive density (ELPD),
+    the ELPD is estimated either by Pareto smoothed importance sampling leave-one-out
+    cross-validation (loo) or using the widely applicable information criterion (WAIC).
+    We recommend LOO.

```

Comment:
We recommend LOO in line with the theory presented by Vehtari et al. (2016) available here: https://arxiv.org/abs/1507.04544.